esp/mlrs-wireless-bridge/mlrs-wireless-bridge-boards.h

```diff
@@ -137,6 +137,26 @@ GPIO15 = RTC_GPIO13
     #define WIRELESS_PROTOCOL 1 // make UDP the default
 
 
+//-- ELRS Tx Module ESP32C3 backpack
```

Comment:
great
but, please add also a comment in the top of the file ;)

Reply:
just this is missing from my side

not sure I phrased well: there should be also a section in the top like that in lines 11-16, just for the new C3 backpack